bench/src/main/scala/benchmarks/HashBench.scala

```diff
@@ -23,8 +23,7 @@ class HashBench {
   val situations = games.flatMap(_.moves).flatMap(_.move).map(_.situationAfter)
 
   @Benchmark
-  def hashes(): Boolean =
+  def hashes() =
     situations.foreach(Hash(_))
-    true
```

Comment:
we should avoid return constant in our bench. Because jvm can do some "constant folding" optimization during benchmarking.

reference: https://www.baeldung.com/java-microbenchmark-harness#constant-folding

I also think I need to read a bit more about benchmarking to make sure what we do with the benchmark is correct. Because it is so easy to be misleading.